packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -668,6 +679,60 @@ class Auth {
         return await this.loginGuestUser()
     }
 
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: authorizeIDP.
+     *
+     */
+    async authorizeIDP(parameters: Parameters<Helpers['authorizeIDP']>[1]) {
+        const privateClient = !!this.clientSecret
+        const redirectURI = this.redirectURI
+        const usid = this.get('usid')
+        const {url, codeVerifier} = await helpers.authorizeIDP(
+            this.client,
+            {
+                redirectURI,
+                hint: parameters.hint,
+                ...(usid && {usid})
+            },
+            privateClient
+        )
+        window.location.assign(url)
+        this.set('code_verifier', codeVerifier)
+    }
+
+    /**
+     * A wrapper method for commerce-sdk-isomorphic helper: loginIDPUser.
+     *
+     */
+    async loginIDPUser(parameters: Parameters<Helpers['loginIDPUser']>[2]) {
```

Comment:
Same comment here as before, as a reviewer it's hard to understand the signature of this function if we are purely relying on typescript, but using a inline type like that which doesn't give me much information.